static/data/community.ts

```diff
@@ -2,7 +2,7 @@ import { MEETING_URL } from './global';
 const header = {
   title: 'Community',
   subtitle:
-    'We want your feedback, issues, patches, and involvement in the development of Podman. **Chat** with us on Slack, IRC, or on our **mailing list**. Submit **issues & pull requests** (see our [CONTRIBUTING guide](https://github.com/containers/podman/blob/main/CONTRIBUTING.md) on how.) Participate in one of our twice-monthly community meetings. You are welcome in our community!',
+    'We want your feedback, issues, patches, and involvement in the development of Podman. **Chat** with us on Slack, Discord, or on our **mailing list**. Submit **issues & pull requests** (see our [CONTRIBUTING guide](https://github.com/containers/podman/blob/main/CONTRIBUTING.md) on how.) Participate in our monthly community meetings. You are welcome in our community!',
```

Comment:
should also mention matrix if we mention discord.

But I am a bit our of the loop with this change, when I brought this topic up it seemed nobody (besides maybe me) is actually engaging in the community chat rooms (matrix/discord/IRC) so I feel like the whole sentence is kinda wrong as it is totally unclear who "us" is. I would think maintainers.